docs/docs/dev_docs/getting_started/updating.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Updating
+---
+
+There are 4 components whose versions need to be kept compatible:
+1. Aztec Sandbox,
+2. Aztec CLI,
+3. Noir compiler `nargo`,
+4. Noir framework for Aztec contracts `aztec.nr`.
+
+## Updating Aztec Sandbox
+To update the sandbox to the latest version, simply run the curl command we used for installation again:
+```shell
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+It will download and start the latest version of sandbox.
+
+If you would like to use a fixed version of the sandbox, you can export the `SANDBOX_VERSION` environmental variable.
+If you are unsure what version to use go to [aztec-packages repository](https://github.com/AztecProtocol/aztec-packages/releases) and choose the `aztec-packages` release based on the changelog.
+
+Then set the `SANDBOX_VERSION` environmental variable to the version you want to use. E.g.:
+```shell
+export SANDBOX_VERSION=0.7.7
```

Comment:
WDYT about showing here the latest value? We could extend the load-versions plugin to also load the current packages version from the `.release-please-manifest.json` on the project root, and define a new component in `docs/src/components/Version/index.js` to show it, as we're doing with the nargo version.

https://github.com/AztecProtocol/aztec-packages/blob/18fe88aa63e9a93f4f62789e94047edf33620bfa/docs/docusaurus.config.js#L78-L104

Reply:
See https://github.com/AztecProtocol/aztec-packages/pull/2440